deployment/docker-compose/arthur-engine/docker-compose.yml

```
@@ -21,13 +21,15 @@ services:
     depends_on:
       - db
     ports:
-      - "3000:3000"
+      - "3030:3030"
     environment:
       - POSTGRES_USER=postgres
       - POSTGRES_URL=host.docker.internal
```

```suggestion
      - POSTGRES_URL=db
```
this should also use the docker internal address

---

I thought this was the situation where we needed to use host.docker.internal, but I will test it again with this change.